ballerina/sql_client.bal

```diff
@@ -26,6 +28,15 @@ public client class SQLClient {
     private string[] keyFields;
     private map<JoinMetadata> joinMetadata;
 
+    # Initializes the SQLClient.
```

Comment:
```suggestion
    # Initializes the `SQLClient`.
```

Reply:
Fixed in [83ba8ac](https://github.com/ballerina-platform/module-ballerina-persist/pull/29/commits/83ba8aca943aaaaffd88f25e28fe62a5c00c1ef6)